app/src/androidTest/java/com/github/sdpcoachme/profile/ProfileActivityTest.kt

```diff
@@ -303,6 +293,49 @@ class ProfileActivityTest {
         }
     }
 
+    @Test
+    fun editSports() {
```

Comment:
Maybe, you could change the name of the test to something a bit more meaningful e.g., "choosingAllSports..."